packages/components/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@backstage/components",
```

Comment:
I think it makes sense to call this `@backstage/core-components`, unless we want people to depend on this package by itself, which I don't think we do